tcms/locale/en/LC_MESSAGES/django.po

```diff
@@ -702,7 +704,11 @@ msgid "Detailed information about test execution status"
 msgstr ""
 
 #: tcms/telemetry/templates/telemetry/include/filters.html:40
-msgid "Test Plan"
+#: tcms/testplans/templates/testplans/search.html:111
+#: tcms/testruns/templates/testruns/get.html:38
+#: tcms/testruns/templates/testruns/mutable.html:59
```

Comment:
> __[Secret Scan]__ It may be dangerous to commit the AWS secret access key. ([view](https://sider.review/gh/repos/92381207/pulls/2528#issue-6063098))
>
> <details>
>
> AWS secret access key is an important credential that allows you to use AWS programmatically.
> Once this leaks, an attacker can access and operate your AWS resources.
>
> See also:
> * https://docs.aws.amazon.com/general/latest/gr/aws-access-keys-best-practices.html
>
> | Rule | Severity |
> | ---- | -------- |
> | `sider.secrets.aws.secret_access_key` | `warning` |
>
> You can [close this issue](https://sider.review/gh/repos/92381207/pulls/2528#issue-6063098) if no need to fix it. [Learn more](https://help.sider.review/getting-started/working-with-issues).
>
> </details>

I closed this issue. Reason:
- When this is not an AWS secret access key.